go/controller/internal/autogen/autogen_api_translator.go

```diff
@@ -413,15 +417,16 @@ func (a *apiTranslator) translateTaskAgent(
 		return nil, err
 	}
 
+	agentName := convertToPythonIdentifier(team.Name)
 	return &api.Component{
 		Provider:      "kagent.agents.TaskAgent",
 		ComponentType: "agent",
 		Version:       1,
-		Label:         "society_of_mind_agent",
+		Label:         agentName,
```

Comment:
note that you'll have to update the UI as well as I am checking for this specific name in order to filter it out from the messages.

Reply:
I think I fixed this in: https://github.com/kagent-dev/kagent/pull/239/commits/77faf401f29b64ab0a5cc7eb4c151f1d21405003